client/src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import hanabiReducer from './hanabiReducer';
```

Comment:
I believe that we should call this `import game from './gameReducer';` and then on line 4 we can just use 

```javascript 
const rootReducer = combineReducers({
  game
})
```